TeamViewerPS/Public/Remove-TeamViewerUserGroupMember.ps1

```diff
@@ -0,0 +1,101 @@
+function Remove-TeamViewerUserGroupMember {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    param(
+        [Parameter(Mandatory = $true)]
+        [securestring]
+        $ApiToken,
+
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserGroupId } )]
+        [Alias("UserGroupId")]
+        [Alias("Id")]
+        [object]
+        $UserGroup,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByUserGroupMemberId")]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserGroupMemberMemberId } )]
+        [Alias("UserGroupMemberId")]
+        [Alias("MemberId")]
+        [object]
+        $UserGroupMember,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByUserId")]
+        [ValidateScript( { $_ | Resolve-TeamViewerUserId } )]
+        [Alias("UserId")]
+        [object]
+        $User,
+
+        [Parameter(Mandatory = $true, ParameterSetName = "ByMembers", ValueFromPipeline = $true)]

```

Comment:
What is the `-Member` parameter do? Can't this be just an `Alias` for `-UserGroupMember`? And `-UserGroupMember` could be `object[]`